pkgs/tools/misc/sfeed/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  pname = "sfeed";
+  version = "0.9.20";
+
+  src = fetchgit {
+    url = "git://git.codemadness.org/sfeed";
+    rev = version;
+    sha256 = "17bs31wns71fx7s06rdzqkghkgv86r9d9i3814rznyzi9484c7aq";
+  };
+
+  installPhase = ''
+    mkdir $out
+    make install PREFIX=$out
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://codemadness.org/sfeed-simple-feed-parser.html";
+    description = "a RSS and Atom parser (and some format programs)";
```

```suggestion
    description = "RSS and Atom parser (and some format programs)";
```

Sorry, for the nitpicking. Description should start with a [capital letter](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes).

---

This is not a nit and required by the contributing guide.